shell/testing/tester_main.cc

```diff
@@ -513,6 +524,94 @@ int RunTester(const flutter::Settings& settings,
   return completion_observer.GetExitCodeForLastError();
 }
 
+#ifdef _WIN32
+#define EXPORTED __declspec(dllexport)
+#else
+#define EXPORTED __attribute__((visibility("default")))
+#endif
+
+extern "C" {
+EXPORTED Dart_Handle LoadLibraryFromKernel(const char* path) {
+  std::shared_ptr<fml::FileMapping> mapping =
+      fml::FileMapping::CreateReadOnly(path);
+  if (!mapping) {
+    return Dart_Null();
+  }
+  return DartIsolate::LoadLibraryFromKernel(mapping);
+}
+
+EXPORTED Dart_Handle LookupEntryPoint(const char* uri, const char* name) {
+  if (!uri || !name) {
+    return Dart_Null();
+  }
+  return Dart_GetField(Dart_LookupLibrary(Dart_NewStringFromCString(uri)),
+                       Dart_NewStringFromCString(name));
+}
+
+EXPORTED void Spawn(const char* entrypoint, const char* route) {
+  auto shell = g_shell->get();
+  fml::AutoResetWaitableEvent latch;
+  auto isolate = Dart_CurrentIsolate();
+  Dart_ExitIsolate();
```

Comment:
nit: Move this closer down to where it's needed (the closure allocation here can happen before exiting the isolate)

Reply:
Done